subgraph/packages/subgraph-dufour-channel/schema.graphql

```diff
@@ -0,0 +1,68 @@
+type Account @entity {
+  "account's address"
+  id: ID!
+  "account's public key"
+  publicKey: Bytes
+  multiaddr: [String!]!
+  "channels where the Account is the source"
+  fromChannels: [Channel!]! @derivedFrom(field: "source")
+  "channels where the Account is the destination"
+  toChannels: [Channel!]! @derivedFrom(field: "destination")
+  "number of outgoing channels"
+  fromChannelsCount: BigInt!
```

Comment:
`fromChannelsCount`, `toChannelsCount` and `balance` in `Account` were not updated when in respective Channels events. When querying an "account" entity with a node address, it should return the total number of outgoing channels in `fromChannelsCount`, the total number of incoming channels in `toChannelsCount` and the sum of stakes in outgoing channels in `balance`

Reply:
Thanks ! The `fromChannelCount` and `toChannelCount` were already updated on channel opening and closing. The balance wasn't indeed.
Question: when a channel is opened, the initial amount is not in the `ChannelOpened` event. Should I retrieve it from hash also, or is a ChannelBalanceIncreased event also emitted on channel opening ?